UltraStar Play/Assets/Scenes/SongEditor/Controls/SongEditorVoiceToggle.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+using UniInject;
+using UniRx;
+
+// Disable warning about fields that are never assigned, their values are injected.
+#pragma warning disable CS0649
+
+public class SongEditorVoiceToggle : MonoBehaviour, INeedInjection
+{
+    public int voiceIndex;
+
+    private string voiceName;
+
+    [Inject(searchMethod = SearchMethods.GetComponentInChildren)]
+    private Toggle toggle;
+
+    [Inject]
+    private Settings settings;
+
+    [Inject]
+    private EditorNoteDisplayer editorNoteDisplayer;
+
+    void Start()
+    {
+        voiceName = "P" + (voiceIndex + 1);
```

Comment:
this might not work for P3 -> lyrics should maybe be visible (and editable?) for all voices?

Reply:
In general, the editor should support N voices and all of one should be able to hide /show them individually. However, I wanted to support only two voices for now.
And I did not know about the "P3 means P1 and P2" thing.

Mmmh, I suggest to merge it as it is now, and make another issue to fix it. We should discuss how to handle N voices in general. The list of voices has to be populated dynamically then.

Reply:
Please bear in mind we can't just change ultrastar file format however we like, but instead there is a plethora of other projects parsing the ultrastar file format. There was already a discussion regarding supporting more than two singers, see  https://github.com/performous/performous/issues/235

Reply:
For the song editor, it should be fine to just support P0 to Pn as each voice being a "separate" voice to edit. Could you maybe change the code so it supports P0 to either be the only voice, or it being one of the voices, or it being empty while there is P1 .. Pn?